assets/js/modules/analytics-4/components/widgets/PopularAuthorsWidget.stories.js

```diff
@@ -204,11 +206,17 @@ export default {
 
 				registry.dispatch( CORE_USER ).setReferenceDate( '2020-09-08' );
 				registry.dispatch( MODULES_ANALYTICS_4 ).setSettings( {
-					propertyID: '12345',
+					propertyID,
 					availableCustomDimensions:
 						KEY_METRICS_WIDGETS[ KM_ANALYTICS_POPULAR_AUTHORS ]
 							.requiredCustomDimensions,
 				} );
+				registry.dispatch( MODULES_ANALYTICS_4 ).receiveGetProperty(
+					{
+						createTime: '2014-10-02T15:01:23Z',
```

Comment:
This just needs to be a valid date that is well behind for our GA4 property to not be in a gathering data state. I had actually copied this over from the GA4 fixtures. See:

https://github.com/google/site-kit-wp/blob/14d70dea611e7de142ce3f6c122f9be15bbcae5a/assets/js/modules/analytics-4/datastore/__fixtures__/properties.json#L6

Do we need to change it?

Reply:
Ah that's fine then, I was mostly curious since it seems arbitrary. If the date has some significance, it would be good to make that more explicit so that it's less likely to be broken unintentionally.